src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketsTelemetry.cs

```diff
@@ -12,15 +12,17 @@ internal sealed class SocketsTelemetry : EventSource
     {
         public static readonly SocketsTelemetry Log = new SocketsTelemetry();
 
+        private PollingCounter? _currentOutgoingConnectAttemptsCounter;
         private PollingCounter? _outgoingConnectionsEstablishedCounter;
         private PollingCounter? _incomingConnectionsEstablishedCounter;
         private PollingCounter? _bytesReceivedCounter;
         private PollingCounter? _bytesSentCounter;
         private PollingCounter? _datagramsReceivedCounter;
         private PollingCounter? _datagramsSentCounter;
 
-        private long _outgoingConnectionsEstablished;
-        private long _incomingConnectionsEstablished;
+        private static long _currentOutgoingConnectAttempts;
+        private static long _outgoingConnectionsEstablished;
+        private static long _incomingConnectionsEstablished;
```

Comment:
I'm not understanding why these became static but seemingly inconsistently with the rest of the fields.

Reply:
More of a hint to the caller that the methods that change these are different (don't follow the `if (Log.IsEnabled()) Log.DoSomething()` pattern. I can change them back to non-static.